Submitted By:            BLFS Book <blfs-book@linuxfromscratch.org>
Date:                    2005-01-24
Initial Package Version: 1.1.9
Upstream Status:         Already in CVS
Origin:                  Cdrdao CVS
Description:             Fixes gcdmaster compilation issue with GCC-3.4.x

$LastChangedBy$
$Date$

diff -Naur cdrdao-1.1.9-orig/xdao/Project.h cdrdao-1.1.9/xdao/Project.h
--- cdrdao-1.1.9-orig/xdao/Project.h	2004-02-12 01:13:32.000000000 +0000
+++ cdrdao-1.1.9/xdao/Project.h	2005-01-25 01:22:03.961953520 +0000
@@ -51,14 +51,12 @@
   virtual void createToolbar() = 0;
   virtual void createStatusbar();
   void updateWindowTitle();
-  void saveProject();
   void saveAsProject();
   Gtk::FileSelection *saveFileSelector_;
   void saveFileSelectorOKCB();
   void saveFileSelectorCancelCB();
 
   virtual bool on_delete_event(GdkEventAny *event);
-  virtual void recordToc2CD() = 0;
   virtual void projectInfo() = 0;
 
 public:
@@ -67,7 +65,9 @@
   void readToc(char *name);
   void statusMessage(const char *fmt, ...);
   void tocBlockedMsg(const char *);
-  virtual bool closeProject() = 0;
+  virtual bool closeProject() = 0; 
+  virtual void saveProject();
+  virtual void recordToc2CD() = 0;
   int projectNumber();
   TocEdit *tocEdit();
   gint getViewNumber();
